ajax/libs/slickgrid/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "slickgrid",
+  "version": "2.3.4",
+  "description": "A lightning fast JavaScript grid/spreadsheet",
+  "filename": "slick.core.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/6pac/SlickGrid.git"
+  },
+  "keywords": [
+    "slickgrid",
+    "grid"
+  ],
+  "author": {
+    "name": "Michael Leibman",
+    "email": "michael.leibman@gmail.com"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/mleibman/SlickGrid/wiki",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/6pac/SlickGrid.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "lib/*",
```

Comment:
In lib, the CDNJS didn't hold `jquery.simulate.js`, `jquery.event.drag-2.2.js`,` jquery.event.drag-2.3.0.js`,` jquery.event.drop-2.2.js`, `jquery.event.drop-2.3.0.js`

And I saw some people use lib in their code form this lib. eg. https://github.com/person2865/d3js-practise/blob/76051d072edfe9af0bd6de75eab52f20967e41f7/index.html

so, I think it need this lib.

PS1. For` jquery-simulate` maybe user can use from https://github.com/jquery/jquery-simulate
PS2. For `jquery.event.drag` `jquery.event.drop `maybe user can use from http://threedubmedia.com/ , but in [README ](https://github.com/6pac/SlickGrid/blob/master/README.md) it said this lib updated jquery.event.drag and updated jquery.event.drop to be compatible with jQuery 3